app/views/parent_interface/consent_forms/cannot_consent_responsibility.html.erb

```diff
@@ -14,5 +14,5 @@
 
 <p>
   If you have any questions, please contact the local health organisation by calling
-  <%= @team.phone %>, or email <%= mail_to @team.email %>.
+  <%= @team.phone %><%= " (#{@team.phone_instructions})" if @team.phone_instructions.present? %>, or email <%= mail_to @team.email %>.
```

Comment:
Since we have this logic in two places I think it's worth creating a helper method for it (see the files in `app/helpers` for other similar ones).